calico-cloud_versioned_docs/version-18/image-assurance/understanding-scan-results.mdx

```diff
@@ -55,71 +56,40 @@ To change the CVSS threshold values, note the following:
 - Changes to threshold values take effect immediately and alter the scan results for images already in the system
 - If you are using admission controller policies, changing a value may allow pods in your Kubernetes cluster that were previously being blocked, to now be deployed or vice versa.
 
-**Enable Runtime View**
+## Export results
 
-You can configure your scan settings to directly monitor pods that are running in your Kubernetes clusters.
-When enabled, Runtime View checks image information from the active container images in all of your connected clusters.
-These images are correlated with the images you have recorded in the Image Assurance system.
+From each tab, you can export data or a JSON file with image URLs. Exporting data is based on the images in the list and the current filter selections. CSV table options include:
 
-To enable Runtime View, select the **Enable Runtime View** checkbox in the **Scan Results** tab.
-You can clear the checkbox to disable Runtime View.
-After disabling Runtime View, some runtime information for existing images will remain in the system.
+- **Export one row per image** - export one row for each image with all associated CVEs condensed into a single column.
+- **Export one row for each image and CVE ID** - export a unique image plus CVE combination for each row. For example, if an image has 10 CVEs, 10 rows are created (1 for each CVE).
 
 :::note
 
-If your cluster does not use the default containerd socket path (`/run/containerd/containerd.sock`), then you must change the path to allow the Runtime View service to collect image information.
-
-To update the CRI socket path for a cluster, run the following command:
-
-```bash
-kubectl patch imageassurance default --type='merge' -p '{"spec":{"criSocketPath":"<path-to-cri-socket>"}}'
-```
-See [the installation reference](../reference/installation/api.mdx#image-assurance.operator.tigera.io/v1.ImageAssuranceSpec) for more details on the ImageAssurance CR.
+Images without associated CVEs are not included in the exported data (regardless if they are included by filters).
 
 :::
 
-### Exclude specific vulnerabilities
+## Exclude vulnerabilities from scan results
```

Comment:
I think we should be a bit more specific in this case, because depending on what you mean by scan results, this statement could be inaccurate. 

The root of the problem is that scan results could potentially mean different things: 
1. The total payload of data that is outputted from running our scanner on an image. 
2. The final verdict (Pass/Warn/Fail) that is generated by our system, based on the max CVSS score for a given image. 

I'm worried that this term is overloaded, such that saying CVEs are excluded from scan results, might give users the wrong impression. 

I personally have resorted to saying: 
1. Scan results 
2. Scan result verdict

I think it's worth building up some consensus with you on this point.

Reply:
How about this? 

## Exclude vulnerabilities from score results

Reply:
This one is a bit tricky. I'm uncertain what the best term could be, but I think "image scoring" could be a bit confusing. 

There's the chance a user conflates the "score" part of the term with the CVSS score that each CVE has. I think score would imply that the value should be numeric. 

Plus I feel like we should still have "scan" in there somehow. The Pass/Warn/Fail assessment is derived from the image scan and provides a verdict or ruling on the whether the image is vulnerable (according the user's threshold values). 

It might be good to pull in @tigeraphil to review this term. 

Off the top of my head, what about something like: 
- Scan verdict 
- Scan assessment
- Scan ruling

Reply:
I think Scan assessment is good enough. Ruling and verdict are too legalistic. 

## Exclude vulnerabilities from image scan assessment

You can exclude specific CVEs from the image scan assessment (for example, a false positive). You can exclude a CVE for an image version, all images in the same repo, or all images in the image score results.

Reply:
Changed to: 
You can exclude specific CVEs from the image scan assessment (for example, a false positive). You can exclude a CVE for an image version, all images in the same repo, or all images in the system.